apps/omg_eth/test/fixtures.exs

```diff
@@ -56,12 +56,12 @@ defmodule OMG.Eth.Fixtures do
     {:ok, true} =
       Ethereumex.HttpClient.request("personal_unlockAccount", ["0x6de4b3b9c28e9c3e84c2b2d3a875c947a84de68d", "", 0], [])
 
-    add_exit_queue =
-      RootChainHelper.add_exit_queue(1, @eth, %{
-        plasma_framework: Encoding.from_hex("0xc673e4ffcb8464faff908a6804fe0e635af0ea2f")
-      })
+    if RootChainHelper.has_exit_queue(1, @eth, contract.contract_addr) do
```

Comment:
Not sure if I read this right... If RootChainHelper has an exit queue, add another one?